config.json

```diff
@@ -0,0 +1,20 @@
+{
+    "Mador" : {
```

Comment:
This config file shouldn't be here. This is supposed to be a db and not part of the code. What if someone else wants to use the bot? Why should they have all of the info of our section. You can maybe add an example file, but using a db like mongo db will solve the need for this too.

Reply:
Noted. Will implement.